addons/project/views/project_views.xml

```diff
@@ -518,11 +518,8 @@
                                 </group>
                             </group>
                             <group>
-                                <group name="group_tasks_managment" string="Tasks Management" col="1" class="row mt16 o_settings_container" groups="project.group_subtask_project,project.group_project_task_dependencies,project.group_project_milestone,project.group_project_recurring_tasks">
+                                <group name="group_tasks_managment" string="Tasks Management" col="1" class="row mt16 o_settings_container" groups="project.group_project_task_dependencies,project.group_project_milestone,project.group_project_recurring_tasks">
                                     <div>
-                                        <setting id="subtask_settings" help="Split your tasks to organize your work into sub-milestones" groups="project.group_subtask_project">
-                                            <field name="allow_subtasks"/>
-                                        </setting>
```

Comment:
Make the setting element, invisible instead of removing it. 🙂 

Because a custo could do a xpath on it, so it is better to make the element invisible instead of removing it 🙂

Reply:
Done